Mathlib/MeasureTheory/Function/ConvergenceInMeasure.lean

```diff
@@ -88,6 +123,23 @@ theorem congr_right (h : g =ᵐ[μ] g') (h_tendsto : TendstoInMeasure μ f l g)
     TendstoInMeasure μ f l g' :=
   h_tendsto.congr (fun _ => EventuallyEq.rfl) h
 
+-- Convergence in measure is stable under taking subsequences.
+theorem sub_TendstoInMeasure' {f : ι → α → E}  {g : α → E} {u v: Filter ι} (huv : v ≤ u)
+    (hg :  TendstoInMeasure μ f u g) : TendstoInMeasure μ f v g :=
+  fun ε hε => Tendsto.mono_left (hg ε hε) huv
```

Comment:
Isn't this lemma a duplicate of the above?

Reply:
Yes, you are right. I deleted them. Thanks!